unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/IWorldState.cs

```diff
@@ -2,13 +2,15 @@
 using System.Collections.Generic;
 using DCL.Controllers;
 using DCL.Models;
+using UnityEngine;
 
 namespace DCL
 {
     public interface IWorldState : ISceneHandler, IService
     {
         HashSet<string> readyScenes { get; set; }
         Dictionary<string, IParcelScene> loadedScenes { get; set; }
+        Dictionary<Vector2Int, string> loadedScenesByCoordinate { get; set; }
```

Comment:
I'm a bit concerned about this 2 collections not being synced. Maybe we should add/remove/update values there in a method and avoid direct access.

Reply:
yeah I was about to make them private so we have proper get and sets but that was going to be a big refactor, if you agree we can do it :)